### ISSUE TITLE
Document minimum ruby version, and check at runtime

### ISSUE THREAD

**Issue by mblakele** [marklogic/roxy#122]
Commit 66d9d1189a1789324fcaefedec0cf0c1bf5775ad introduced a dependency on ruby 1.9 or later. That isn't a problem in itself, but should be documented as a requirement in the README. Right now it just says "ruby".

For bonus points, the `ml` command should check the ruby environment and warn if it doesn't appear to meet requirements.

The main use case is that OS X 10.8 still preinstalls ruby 1.8, and developers may not realize that they need 1.9 or later. The fix is easy: `brew install ruby` - and OS X 10.9 should come with ruby 2.0. This could also come up when deploying to older linux machines or machine images, in various environments.

**Comment by grtjn** [comment]
Yeah, we should either fix that, or update the message. Reopening this issue. Sorry for the late reply..
